types/rdf-js/index.d.ts

```diff
@@ -145,7 +146,7 @@ export interface DefaultGraph {
  * @see BlankNode
  * @see Variable
  */
-export type Quad_Subject = NamedNode | BlankNode | Variable;
+export type Quad_Subject = NamedNode | BlankNode | Quad | Variable;
```

Comment:
Why not `BaseQuad`?

Reply:
`Quad_Subject` is aimed to be used in the standard (non-generalized) RDF representation `Quad`. Allowing generalized RDF inside non-generalized RDF does not make much sense to me, so I would keep them strictly separate.